Client/src/Modelo/main.java

```diff
@@ -0,0 +1,28 @@
+package Modelo;
+
+public class main
```

Comment:
```suggestion
public class Main
```
Para respetar la nomenclatura determinada de las clases en Java. Comienzan siempre con mayúscula